eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -195,7 +195,8 @@ jobs:
 
   - script: |
       make az_core_cov_xml
-      make az_iot_cov_xml
+      make az_iot_hub_cov_xml
```

Comment:
How do we avoid this from getting out of sync? Do we need to doc this somewhere, put a comment, etc. so that if someone decides to rename it, they know to update this script as well.

Reply:
We could think about somehow hooking it into cmake and let cmake do it. Where if we want to build it, set some cmake flag and that way it's self contained in cmake.